kubernetes/linux/main.sh

```diff
@@ -855,6 +855,23 @@ else
 fi
 source ~/.bashrc
 
+# manually set backpressure value using container limit only when neither backpressure or fbit tail buffer is provided through configmap
+if [ -n "${BACKPRESSURE_THRESHOLD}" ]; then
+      echo "Setting MDSD backpressure threshold from configmap"
+      export MDSD_BACKPRESSURE_MONITOR_MEMORY_THRESHOLD_IN_MB=${BACKPRESSURE_THRESHOLD}
```

Comment:
potential mismatch in the units (left and right), left is MB and we dont know whats the units on the right hand side. ensure we have consistent units on both sides to avoid any manual errors.

Reply:
addressed